go/extractor/project/project.go

```diff
@@ -49,53 +48,47 @@ type GoWorkspace struct {
 }
 
 // Represents a nullable version string.
-type GoVersionInfo struct {
-	// The version string, if any
-	Version string
-	// A value indicating whether a version string was found
-	Found bool
-}
+type GoVersionInfo = util.SemVer
```

Comment:
We're now combining what used to be two different states: "found a go version, but it was invalid" and "didn't find a go version". Is it okay that we can't distinguish those any more?

Reply:
I think I checked to see if there were any cases where we cared about that distinction, but I'll check again to make sure.